### PR TITLE
TST: add Pluto datasets in polar and spherical coordinate

### PR DESCRIPTION
**Pull request by xshaokun**
Two new Pluto datasets:

- pluto_disk_planet: 3D HD disk-planet interaction problem in spherical coordinate
- pluto_rotor: 2D MHD rotor problem in polar coordinate

`test_pluto_wrong_definitions_header_with_geometry()` was modified only for making the datasets pass the test, while this test will be removed in future commit.

Due to #83 , the unit definitons in pluto_disk_planet are given by exact value for now.

### CODE REVIEW

File: tests/data/pluto_rotor/vtk.out [hunk 1]
```diff
@@ -0,0 +1,31 @@
+0 0.000000e+00 1.000000e-06 0 single_file little rho vx1 vx2 vx3 Bx1 Bx2 Bx3 prs
```
Comment (neutrinoceros) [inline tests/data/pluto_rotor/vtk.out:1]: Just curious, does the "little" column refer to endianness ? If so, we may have a bug to fix for the case big-endians are used. Idefix _always_ outputs in little endian so it's currently hard-coded in the frontend

**Review by neutrinoceros — APPROVED** [review]
Just a quick question that's not directly linked to this particular PR.

Thanks a lot for doing this, it'll be extremely valuable for my refactor PR !

File: tests/data/pluto_rotor/vtk.out (hunk 1 above)
Comment (xshaokun) [reply tests/data/pluto_rotor/vtk.out:1]: Yes, it refers to endianness.

Comment (xshaokun) [reply tests/data/pluto_rotor/vtk.out:1]: But I’ve never changed the endianness before, so let me check it tomorrow to figure out how to specify it in PLUTO.

Comment (neutrinoceros) [reply tests/data/pluto_rotor/vtk.out:1]: Sure, I'll open an issue so we don't loose this thread !

**Pull request merged**